jgiven-core/src/main/java/com/tngtech/jgiven/exception/AmbiguousFormattingException.java

```diff
@@ -1,12 +0,0 @@
-package com.tngtech.jgiven.exception;
```

Comment:
Depending on how strongly you feel about backwards compatibility, we would have to keep this class even if now unused.